mongodb/intro/aggregation-ii/aggregation-sortbycount-ii.md

```diff
@@ -0,0 +1,97 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - deep
+  - workout
+
+type: normal
+
+category: tip
+
+---
+
+# Aggregating with $sortByCount and $match
+
+---
+## Content
+
+Previously we used only `$sortByCount`, now we can use the `$match` aggregation stage to only match documents of a certain `type` and perform the `$sortByCount: "$secondType"` for documents that match our specified `type`.

```

Comment:
We should get out of the habit of using "Previously..." or "In the previous insight..." from now on (when writing new insights or when fixing old insights). We should try to make insights as individual as possible. This might not be possible in some cases, but there we can quickly sumarize the concept either in a phrase or a footnote.

In this case, we can just entirely skip the "Previously.." introduction:
```suggestion
We can use the `$match` aggregation stage to only match documents of a certain `type` followed by sorting by count for documents that match our specified `type`.
```